src/sentry_value.h

```diff
@@ -61,6 +61,13 @@ sentry_value_t sentry__value_new_list_with_size(size_t size);
  */
 sentry_value_t sentry__value_new_object_with_size(size_t size);
 
+/**
+ * Constructs a new Span.
+ *
```

Comment:
```suggestion
 *
 * Borrows `parent`.
```

Reply:
as noted in https://github.com/getsentry/sentry-native/pull/631#issuecomment-999960844, i think it would be fine to not mention places where parameters are borrowed, as that's the assumed default.